renovate.json

```diff
@@ -47,6 +47,11 @@
       "matchPackageNames": ["@unhead/vue"],
       "schedule": ["on the 3rd day of the month"]
     },
+    {
+      "matchPackageNames": ["@vueuse/core"],
+      "automerge": true,
```

Comment:
I am happy with automerging as the localStorage tests should be comprehensive enough to catch any problems